apps/arena/lib/arena/configuration.ex

```diff
@@ -138,4 +150,63 @@ defmodule Arena.Configuration do
       skill
     end
   end
+
+  ## Why do we even need this? Well it happens that some of our fields are represented
+  ## as Decimal. To prevent precision loss this struct its converted to a string of the float
```

Comment:
Can't we just change the Decimals to Floats instead?

Reply:
Yes, but we can't have the embedded maps because there is a sort of bug/issue with Ecto and Jason in which Floats end up been parsed back as integers (4000.0 as 4000) and that would crash while running the game cause Rust expects floats

So how about we tackle it in the follow up to fix the whole embedding situation?

Reply:
Yeah let's do that